css/home.css

```diff
@@ -22,6 +75,7 @@
   margin-left: 5%;
   position: absolute;
   bottom: 10%;
+  width: 100%;
```

Comment:
why are you adding a `width` here?

Reply:
that is not mine.

Reply:
please remove it, as gh thinks it is yours

Reply:
done.